NEWS.md

```diff
@@ -14,11 +14,15 @@
 * Added Dao components for module schema.
 * Added DAO component for tenant schema
 * Added API for Event Types managing
+* Added API for Publishers managing
 
- | METHOD |             URL                                | DESCRIPTION                                      |
- |--------|------------------------------------------------|--------------------------------------------------|
- | GET    | /pubsub/event-types                            | Get collection of Event Descriptors              |
- | POST   | /pubsub/event-types                            | Create new Event Type                            |
- | GET    | /pubsub/event-types/{eventTypeName}            | Get Event Descriptor of particular event type    |
- | PUT    | /pubsub/event-types/{eventTypeName}            | Update Event Descriptor of particular event type |
- | DELETE | /pubsub/event-types/{eventTypeName}            | Delete event type                                |
+ | METHOD |             URL                                                                         | DESCRIPTION                                      |
+ |--------|-----------------------------------------------------------------------------------------|--------------------------------------------------|
+ | GET    | /pubsub/event-types                                                                     | Get collection of Event Descriptors              |
+ | POST   | /pubsub/event-types                                                                     | Create new Event Type                            |
+ | GET    | /pubsub/event-types/{eventTypeName}                                                     | Get Event Descriptor of particular event type    |
+ | PUT    | /pubsub/event-types/{eventTypeName}                                                     | Update Event Descriptor of particular event type |
+ | DELETE | /pubsub/event-types/{eventTypeName}                                                     | Delete event type                                | 
+ | POST    | pubsub/event-types/declare/publisher                                                   | Create publisher                                 |
+ | DELETE  | pubsub/event-types/declare/publisher?moduleName={moduleName}&eventType={eventTypeName} | Delete publisher declaration                     |
```

Comment:
Is there a good reason to have word 'declare' in the naming of endpoints? It's a verb and in Rest the action is expressed by the method (POST and DELETE in this case). In addition, we tend to have plural names for endpoints. Couldn't it be pubsub/event-types/publishers or smth for POST? As for DELETE, why event type is passed as parameter and not in the name of the resource as it is for GET method? Could it be pubsub/event-types/{eventTypeName}/publishers?moduleName={moduleName}?

Reply:
Regarding “declare”, so far we decided to leave it as it is and in the future with the author of pub-sbu rfc choose a more convenient word. As for endpoint variant POST pubsub/event-types/publishers, it will have been overridden by /pubsub/event-types/{eventTypeName}. Also, singular names used to avoid overriding by /pubsub/event-types/{eventTypeName}/publishers. 
Passing an event type using a path variable to DELETE publisher would be better, I will make the appropriate changes.